runelite-client/src/main/java/net/runelite/client/plugins/skillcalculator/SkillCalculator.java

```diff
@@ -42,10 +42,8 @@
 import javax.swing.JCheckBox;
 import javax.swing.JLabel;
 import javax.swing.JPanel;
-import net.runelite.api.Client;
-import net.runelite.api.Experience;
-import net.runelite.api.Skill;
-import net.runelite.api.VarPlayer;
+
+import net.runelite.api.*;
```

Comment:
[Our code conventions are to avoid star imports.](https://github.com/runelite/runelite/wiki/Code-Conventions) Revert this change.